.github/workflows/test-accessibility-alt-text-bot.yml

```diff
@@ -5,11 +5,11 @@ on:
   pull_request:
     types: [opened, edited]
   issue_comment:
-    types: [created, edited]
+    types: [created, edited, deleted]
   discussion:
     types: [created, edited]
   discussion_comment:
-    types: [created, edited]
+    types: [created, edited, deleted]
```

Comment:
Is this in place so that the alt text comment by the bot can be deleted if the relevant comment is deleted?

EDIT: ooh yeah looks like it!

Reply:
If a consumer upgrades to the new version but forgets to add this, will the action start failing, or will it fail silently? 

If it's a breaking change, I wonder if we'd want to bump to major version with a note 🤔.

Let's also update the config in the README!

Reply:
It shouldn't fail! They just won't get their comments deleted. (I tested this!) AND GOOD CALL i'll add this to the README.